app/controllers/events_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class EventsController < ApplicationController
+  def new
+    @event = Event.new
+  end
+
+  def create
+    user = User.find_by(id: current_user)
+
+    @event = user.events.build(event_params)
```

Comment:
You should set the connected user into `current_user` and create the user with `current_user.events.build(...)`